lib/index.js

```diff
@@ -5,10 +5,16 @@ const fs = require('fs')
 const yargsParser = require('yargs-parser')
 
 const { randomChars, resolveFromModule, resolveFromRoot } = require('./utils')
-
-const args = yargsParser(process.argv.slice(2))
-
-const filesToCheck = args._.filter(file => /\.(ts|tsx)$/.test(file))
+const argvCleaned = process.argv.slice(2)
+const args = yargsParser(argvCleaned)
+const filesToCheck = args._.filter(file => /\.(ts|tsx)$/.test(file)) // but what about globs ?
```

Comment:
Do you believe we should use glob instead of regex here? What would be the benefit?

Reply:
No , it's just that the file argument to the tsc cli can be a glob, as the example [here](https://www.typescriptlang.org/v2/docs/handbook/compiler-options.html#using-the-cli) shows:
```bash
tsc src/*.ts
```
And we should detect them because they should not be put in the `files` property of the temporary tsconfig, but in the `include` property.
I left that for later, but I put a comment to at least have the maintainers be aware of the potential issue ;-).